librz/include/rz_core.h

```diff
@@ -200,6 +200,49 @@ typedef struct {
 	char *cmd;
 } RzCoreGadget;
 
+/**
+ * \brief Options for controlling address description output
```

Comment:
```suggestion
 * \brief Options for controlling address description output.
```

Please be more pedantic with spelling and grammar in API docs.